tests/fixtures/models/store.py

```diff
@@ -15,14 +16,164 @@
 
 import pytest
 
+from ...utils import update_store, create_store
+
+
+DEFAULT_STORE_UNITS_1 = [
+    ("Unit 1", "Unit 1"),
+    ("Unit 2", "Unit 2")]
```

Comment:
Ending `]` in its own line please. The same applies everywhere. Also for dicts and function and method calls.